test/lit/wasm-split/basic.wast

```diff
@@ -0,0 +1,141 @@
+;; RUN: %wasm-split %s --export-prefix='%' -o1 %t.none.1.wasm -o2 %t.none.2.wasm -v 2>&1 \
```

Comment:
Haven't read the PR yet, but are we introducing lit tests here? Are we supposed to add tests in this format or the current txt format from now? Are we planning to eventually migrate all existing tests to lit tests? Maybe I missed some previous discussions on this.

Reply:
We’ve thought about it before, but you haven’t missed any recent discussions or decisions. I just wanted to see what would be possible. I expect we will discuss this and try to answer all those questions together on Monday. I think it would be nice to move to lit and FileCheck for most of our tests, but I don’t know what anyone else thinks about that.